thread/common/omrthread.c

```diff
@@ -2071,9 +2091,12 @@ threadFree(omrthread_t thread, int globalAlreadyLocked)
  * Exit from the current thread.
  *
  * If the thread has been detached it is destroyed.
+ *
+ * @param[in] globalAlreadyLocked indicates whether the thread library global mutex is already locked
+ *            Also implies that omrthread_tls_finalize was called before the global mutex was locked
  */
```

Comment:
missing documentation: `@return none` or `return void`

Reply:
It was like that when I got here :)